### PR TITLE
merge C++ and Fortran unit tests

### PR DESCRIPTION
**Pull request by zingale**
The merges the test_eos unit tests

### CODE REVIEW

**Comment by zingale** [comment]
addresses #266

**Comment by zingale** [comment]
okay, this works, and the benchmarks for the Fortran and C++ are the same as before, but they are just named differently so the suite will complain.  But once this is merged, it'll all be happy.

File: unit_test/test_eos/input_eos [hunk 1]
```diff
@@ -1,4 +1,4 @@
 n_cell = 16
 max_grid_size = 32
 
-amr.probin_file = probin
+amr.probin = probin
```
Comment (maxpkatz) [inline unit_test/test_eos/input_eos:4]: `amr.probin_file` is the correctname of the parameter, what's up here?

Comment (zingale) [reply unit_test/test_eos/input_eos:4]: you're right -- that was a typo, but since the default is "probin", it was still getting the right probin file

**Comment by maxpkatz** [comment]
> 
> 
> okay, this works, and the benchmarks for the Fortran and C++ are the same as before, but they are just named differently so the suite will complain. But once this is merged, it'll all be happy.

We could keep the plotfile names the same, couldn't we?

**Comment by zingale** [comment]
yeah, I could keep them the same -- I was trying to follow the convention used in test_react and the other unit tests.  But I'll make them have the same name as before for now and then we can sync things up later.

**Pull request merged**